src/components/Activities/Definition.tsx

```diff
@@ -0,0 +1,9 @@
+interface Activities {
```

Comment:
The convention for types is to use the singular `Activity`, please rename 🙏

Reply:
Done in 6f5ba69